### ISSUE TITLE
GTFS Schedule: Consider storing processed data as JSON rather than CSV

### ISSUE THREAD

**Issue by lauriemerrell** [cal-itp/data-infra#1352]
Currently, our pipeline (specifically [`gtfs_loader.gtfs_schedule_history_load.py`](https://github.com/cal-itp/data-infra/blob/main/airflow/dags/gtfs_loader/gtfs_schedule_history_load.py)) copies raw GTFS data (stored as .txt files from the unzipped GTFS feed) and converts it to .csv files to be ingested into BigQuery external tables.

The current copy operation **drops** unknown columns and reorders columns; this is because BigQuery does not handle unknown columns in CSV and requires columns be in the same order. This means that if we ever want to load a new column, we would need to **reprocess** all the historical schedule data before we can add it to the BigQuery external table schema.

If instead we converted all GTFS data to JSON (even though it is natively tabular), we could support changing schemas simply by changing the BQ table definition. 

AC for this ticket:

- [ ] Experiment with converting GTFS schedule data to JSON / gzipped JSON:
   * Examine file size implications, particularly for `stop_times`
   * Examine behavior in cases like #1015 - what kind of errors would be thrown when converting corrupt CSVs?
 - [ ] Make a determination whether JSON is viable for GTFS schedule data (storage increase cost / conversion reliability) 
 - [ ] If viable, make a follow-up ticket to change the actual processing to produce JSON

**Comment by lauriemerrell** [comment]
We did this in the rewrite
